addons/jam/magwells_3006.hpp

```diff
@@ -1,5 +1,5 @@
-    class CBA_3006_BAR {};          // M1918 BAR

```

Comment:
What changed here? line ending?

Reply:
That or empty whitespace at the end of lines.

Reply:
Line ending.